### ISSUE TITLE
placeholders are not recognized in env variables

### ISSUE THREAD

**Issue by jchmielniak** [borgbackup/borg#5998]
## Have you checked borgbackup docs, FAQ, and open Github issues?

Yes

## Is this a BUG / ISSUE report or a QUESTION?

Question

#### Your borg version (borg -V).
1.1.17 linux

#### Operating system (distribution) and version.

CentOS 7

## Describe the problem you're observing.
Using placeholders in environment vars seems not to be supported.

F.e.:

```
export BORG_REPO=/data/backup/borg/{hostname}
borg list
Repository /data/backup/borg/{hostname} does not exist.
```
But:
```
borg list /data/backup/borg/{hostname}
```
works.

**Comment by ThomasWaldmann** [comment]
Yeah, guess there is no code for that - but it is an interesting idea!

**Comment by ThomasWaldmann** [comment]
Fixed in 1.1-maint, see PR.
